### DISCUSSION TITLE
OHIF Viewer in Radiology Tutorial Outdated

### DISCUSSION THREAD

**Discussion by johnnyyangdiamond** [Project-MONAI/MONAILabel#1699] [Q&A]
I was following along with the spleen segmentation tutorial using the OHIF Viewer but I noticed that the UI for the OHIF viewer in the tutorial is outdated. I ran into problems with part 6.3 of the tutorial where I could supposedly edit the annotated label. I was able to run the inference model successfully, however, I did not have any options to modify the label like in the tutorial.

![Screenshot 2024-05-24 at 5 34 49 PM (2)](https://github.com/Project-MONAI/MONAILabel/assets/122113507/7676c9d1-fef8-477c-b80d-18eedc022a0b)

The OHIF viewer now has segmentation and MONAI Label as different components.

![Screenshot 2024-05-24 at 5 32 07 PM (2)](https://github.com/Project-MONAI/MONAILabel/assets/122113507/4f354da9-4cbc-4559-84c7-a7fa2d911cd0)

I can access the segmentation portion of the OHIF Viewer, but I cannot edit the inferred annotation created by the model and can only create my own.

![Screenshot 2024-05-24 at 5 37 00 PM (2)](https://github.com/Project-MONAI/MONAILabel/assets/122113507/3d9c2ed3-9ab1-449e-9ce6-b0a48d110632)

**Comment by diazandr3s** [comment]
Hi @johnnyyangdiamond,

The tutorials were created with OHIFv2. Since [version 0.8.0](https://github.com/Project-MONAI/MONAILabel/releases/tag/0.8.0) MONAI Label uses OHIFv3.
If you'd like to use the OHIFv2 as in the tutorials, please install[ MONAI Label version 0.7.0](https://github.com/Project-MONAI/MONAILabel/releases/tag/0.7.0). 

if using PyPI, please use `pip install monailabel==0.7.0`

**Reply by johnnyyangdiamond** [reply]
Thanks for the help! However, I installed version 0.7.0 using PyPi but when I tried to start the server using the command in the tutorial "monailabel start_server --app apps/radiology --studies http://127.0.0.1:8042/dicom-web --conf models segmentation_spleen" I got the error:

from monai.data import write_nifti
ImportError: cannot import name 'write_nifti' from 'monai.data' (/opt/anaconda3/envs/monailabelV0.7.0/lib/python3.9/site-packages/monai/data/__init__.py)

**Reply by diazandr3s** [reply]
Sorry, are you using the radiology app from version 0.7.0? or the one from 0.8.0?
Please make sure you download the one from 0.7.0

**Reply by johnnyyangdiamond** [reply]
I was in fact using the radiology app from a different version. Is there a way to specify the version when downloading the radiology app or do I just use the same command in the tutorial "monailabel apps --download --name radiology --output apps" with the 0.7.0 version?

I did the command in the tutorial which gave me a different error this time:

ImportError: cannot import name 'AddChanneld' from 'monai.transforms' (/opt/anaconda3/envs/monailabelV0.7.0/lib/python3.9/site-packages/monai/transforms/__init__.py)

**Reply by diazandr3s** [reply]
Regarding this:

> Is there a way to specify the version when downloading the radiology app or do I just use the same command in the tutorial "monailabel apps --download --name radiology --output apps" with the 0.7.0 version?

That's correct. If you use that command on version 0.7.0, it'll download the radiology app for this version.

> I did the command in the tutorial which gave me a different error this time:
> 
> ImportError: cannot import name 'AddChanneld' from 'monai.transforms' (/opt/anaconda3/envs/monailabelV0.7.0/lib/python3.9/site-packages/monai/transforms/init.py)

Sorry, can you try on a fresh python env? I think this issue comes from having still some dependencies of the latest MONAI Label.

Let us know,
